build.gradle

```diff
@@ -104,6 +104,15 @@ wrapper {
     gradleVersion '2.5'
 }
 
+task bootRunDev(type: org.springframework.boot.gradle.run.BootRunTask) {
+	group = 'Application'
+	doFirst() {
+		main = project.mainClassName
+		classpath = sourceSets.main.runtimeClasspath
+		systemProperty "APP_ENV", "dev"

```

Comment:
You could use simple strings instead of `GString`'s as these are just constants.